component/registration_form.jsx

```diff
@@ -0,0 +1,44 @@
+import { TYPE_JSON } from '../api/defs'
+
+export const registrationForm = () => {
+  const onSubmit = async (e) => {
+    e.preventDefault()
+    // form data
+    const form = new FormData(e.target)
+    const data = Object.fromEntries(form)
+    const json = JSON.stringify(data)
+    const fetching = await fetch('/api/user/registerForm', {
+      method: 'POST',
+      headers: { ...TYPE_JSON },
+      body: { ...json },
+    })
+    const response = await fetching.json()
+    if (response.status) {
+      // redirect some where
+    } else {
+      // show Errors
+      document.querySelector('span').innerText = response.errors.join(',')
```

Comment:
Good that you didn't use `.innerHTML`, but you should prefer `.textContent` as `innerText` was not  standard, is slow and do not exist on all types of nodes.

I also like `.textContent` more because it doesn't look like `innerHTML` and as such scare me less :)